scripts/deb-builder-helpers.sh

```diff
@@ -335,13 +335,14 @@ build_archive_deb () {
   cp ./default/src/app/extract_blocks/extract_blocks.exe "${BUILDDIR}/usr/local/bin/mina-extract-blocks"
   
   mkdir -p "${BUILDDIR}/etc/mina/archive"
-  cp ../scripts/archive/download-missing-blocks.sh "${BUILDDIR}/etc/mina/archive"
```

Comment:
Are these two scripts equivalent in what they achieve?

Reply:
Yes, With addition that missing-blocks-guardian has more features and is actively used in our infra and by community